app/graphql/gym_trackr_context.rb

```diff
@@ -26,7 +26,11 @@ def current_user_id
   def current_user
     return unless current_user_id
 
-    @current_user ||= User.find(current_user_id)
```

Comment:
Needed to avoid raising a server exception to the frontend in this scenario:
- A request gets into the system
- This request contains a valid not expired JWT
- The user that the JWT references is not found in the database